package.json

```diff
@@ -4,6 +4,8 @@
   "repository": "Vets-Who-Code/vwc-site",
   "dependencies": {
     "babel-preset-gatsby": "0.2.0",
+    "enzyme": "^3.11.0",
```

Comment:
@eprislac  can you please revert/remove enzyme and enzyme-adapter from `package.json` and `setup-test-env.js`